Dockerfile

```diff
@@ -13,13 +13,13 @@ COPY routes /envelope-game/routes
 COPY __tests__ /envelope-game/__tests__
 COPY __mocks__ /envelope-game/__mocks__
 
-RUN npm test
+RUN yarn test
 
-RUN npm run build
+RUN yarn run build
 
-# RUN npm prune --production
+RUN yarn install --production --ignore-scripts --prefer-offline
```

Comment:
This isn't doing the same thing as `npm prune --production`.  honestly, I am not sure if there is a yarn equivalent.  but as-is, this isn't needed.

I really don't want to increase the scope here, but what we should be doing is using the build stage to install all dependencies, and build the bundled react app, then using the final stage to only install production dependencies and copy the build artifact from the build stage.

that can be done in a follow-up PR though

Reply:
gotcha, we'll probably work on that after this PR then